invisible_cities/daemons/idaemon.py

```diff
@@ -0,0 +1,15 @@
+from importlib import import_module
+import traceback
+
+
+def invoke_daemon(daemon_name):
+    """Takes a daemon name and returns an instance of the daemon"""
+    try:
+        module_name = 'invisible_cities.daemons.' + daemon_name
+        daemon_class  = getattr(import_module(module_name),
+                                          daemon_name.capitalize())
+    except ModuleNotFoundError:
+        traceback.print_exc()
```

Comment:
Why do we prefer use of an unknown daemon name to continue running the program while polluting standard output?

Surely the correct behaviour would be to fail, because the configuration contains nonsense.

In other words, the `ModuleNotFound` error should *not* be caught here.

Reply:
The idea is that daemons do chores that you don't want to do in the standard flow of the city (for example control histograms). Since those chores may also change, one has the possibility of invoking different daemons from the same city.

Reply:
I don't see how that addresses the question.

`ModuleNotFound` will be raised if a daemon name is incorrect. In this case we don't want the code to run (while making noise on `stdout`); we want the code to stop, so that the daemon name can be corrected.

Reply:
Sorry didn't get the sense of your question, and BTW I don't remember having written this part of code (probably the few neurones I have left are dying fast). Agree, we want the code to stop.

Reply:
OK, I've fixed this in a miscellaneous cleanup branch I've got here. Progress is slow, as I only have scraps of time here and there.